### DISCUSSION TITLE
Welcome to NotyKT Discussions!

### DISCUSSION THREAD

**Discussion by PatilShreyas** [PatilShreyas/NotyKT#98] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by maulikhirani** [comment]
@PatilShreyas what would be the recommended pathway to dive into learning Ktor?

**Reply by PatilShreyas** [reply]
Currently, I don't have any specific pathway to learn Ktor. But I would say directly work on a project and learn Ktor side by side as it's easy to grasp.

**Comment by deepakkumar-mahindra** [comment]
HI @PatilShreyas, @maulikhirani  I am currently rearchitecturing our android app. I followed this project for some best practices. Here is my one doubt. How edittext restore its text in loginfragment while configuration changes? I am not able to understand this behaviour.
